cpp/benchmarks/io/cuio_common.hpp

```diff
@@ -81,7 +81,8 @@ class cuio_source_sink_pair {
   static temp_directory const tmpdir;
 
   io_type const type;
-  std::vector<char> buffer;
+  std::vector<char> h_buffer;
+  rmm::device_uvector<std::byte> d_buffer;
```

Comment:
Why they are using different data types (`char` vs `std::byte`)? Can we use the same?